packages/desktop-gui/cypress.json

```diff
@@ -2,5 +2,9 @@
   "baseUrl": "http://localhost:5005",
   "projectId": "ypt4pf",
   "viewportWidth": 800,
-  "viewportHeight": 850
+  "viewportHeight": 850,
+  "reporter": "../../node_modules/cypress-multi-reporters/index.js",
```

Comment:
are you pointing at the cypress-multi-reporters at the root, then why add them to each package?

Reply:
it makes intuitive sense for them to be installed per-package, but yarn hoists them, and cypress won't load it by name if it's hoisted outside the current directory (#4536). so it was really just a choice, it could also be installed in the root package